src/jsonrpccpp/common/jsonparser.h

```diff
@@ -10,6 +10,6 @@
 #ifndef JSONRPC_CPP_JSONPARSER_H_
 #define JSONRPC_CPP_JSONPARSER_H_
 
-#include <@JSONCPP_INCLUDE_PREFIX@/json.h>
+#include <json/json.h>
```

Comment:
Do not remove that. It assures that this library can build against old jsoncpp 0.6.x versions.